en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -3,43 +3,41 @@
+        In this quick start guide, let's use the mock service at [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed), which is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You'll see the following JSON message: `{"hello": "world"}`.
  
 ## Step 1: Create an API from scratch
-Follow the procedure below to create an API using the mock REST service you tested in the prerequisites section:
+Follow this procedure to create an API using the mock REST service you tested in the prerequisites section:
 
-1. Access the Choreo Console via [https://console.choreo.dev/](https://console.choreo.dev/).
-2. Sign in using either your GitHub or Google credentials.
-3. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page. 
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
+2. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page. 
     
     !!!note
         If this is not the first API you are creating, you must click **+ Create** to go to the **Create API** page.
 
-4. On the **Create API** page, go to the **Start from scratch card** and click **Next**.
-5. In the **Create API from REST API** form, fill in the details as follows:
+3. On the **Create API** page, go to the **Start from scratch card** and click **Next**.
+4. In the **Create API from REST API** form, fill in the details as follows:
     - **API Name**: `HelloWorld`
     - **API Version**: `1.0.0`
-    - **Endpoint**: http://www.mocky.io/v2/5185415ba171ea3a00704eed
-6. Click **Create**. This displays an overview of the API.
+    - **Endpoint**: `http://www.mocky.io/v2/5185415ba171ea3a00704eed`
+5. Click **Create**. This displays an overview of the API.
 
-       ![API Overview](../assets/img/apis/api-overview.png)
+      ![API Overview](../assets/img/apis/api-overview.png)
        
-7. Under **API Configuration**, click **Resources**.
-8. Follow the procedure below to add a resource to the API:
-    1. Select `GET` as the HTTP verb.
+6. Under **API Configuration**, click **Resources**. Then follow this procedure to add a resource to the API:
+    1. Select **GET** as the HTTP verb.
     2. Enter `hello` as the URI pattern.
     3. Click the **+** icon to add the resource.
-    4. Click **Save**. You can see a confirmation message to proceed with the changes.
+    4. Click **Save**. You'll see a confirmation message to proceed with the changes.
     5. Click **Save** to proceed.
 
 Now you have created the API and added a GET resource to it.
 
 ## Step 2: Deploy the API
-Follow the procedure below to create a revision of the API and deploy it:
+Follow this procedure to create a revision of the API and deploy it:
```

Comment:
Should we mention why we are deploying it? "so that the API can be invoked" ? and also, where the API gets deployed by default?

Reply:
Changed to "Follow this procedure to create a revision of the API and deploy it so that you can try it out:"
Did not mention where the API gets deployed by default because it can be too much information at this point.